src/app/components/CourseChoice/CourseChoise.stories.tsx

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import CourseChoice from "./CourseChoice";
+import PrimPrioIcon from "../../assets/icons/PrimPrioIcon";
+import SecPrioIcon from "../../assets/icons/SecPrioIcon";
+import ThirdPrioIcon from "../../assets/icons/ThirdPrioIcon";
+import BinIcon from "../../assets/icons/BinIcon";
+
+export default {
+  title: "Component/CourseChoice",
+  component: CourseChoice,
+};
+
+export const firstChoice = (): JSX.Element => (
+  <CourseChoice
+    prioIcon={<PrimPrioIcon />}
+    courseChoice={"Tanzen"}
+    binIcon={<BinIcon />}
+  />
```

Comment:
Is it correct that you have two different types of course choices (primary/secondary)?
It would be easier to use if you change it like this:

```tsx
  <CourseChoice
    type="primary"
    value="Tanzen"
  />
```

```tsx
  <CourseChoice
    type="secondary"
    value="Basketball"
  />
```